### PR TITLE
Fixed Incoherent notation for the RZX Gate api doc

### LINKED ISSUE #12684: Incoherence in notation for RZX gate
### URL to the relevant documentation

https://docs.quantum.ibm.com/api/qiskit/qiskit.circuit.library.RZXGate

In examples section, third equation

### Select all that apply

- [X] typo
- [ ] code bug
- [ ] out-of-date content
- [ ] broken link
- [ ] other

### Describe the fix.

If we follow the notation introduce for RZX(θ) q0,q1=exp(−i θ/2 X⊗Z) which are the default notation, it would mean that in the "examples" section, we should have RZX(θ=π)=−iX⊗Z instead of −iZ⊗X. 
It's not coherent with the fact that in the note section, we use big-endian convention for the tensor notation.
Or else we have to tell in which way we apply the gate by adding the q1,q0 index like this: RZX(θ=π)q1,q0=−iZ⊗X.

### PR DESCRIPTION
**Pull request by Ameya28Bhave**
### Summary
Fixes #12684 

Fixed the issue of notation for the RZX gate. The example section of the api document has been updated and follows the big-endian format. For example now the notation is showed as $`R_{ZX}(\theta = 0)\ q_0, q_1 = I`$ .

### CODE REVIEW

File: releasenotes/notes/incoherence-notation-rzxgate-16284-df215ab4903f5e3e.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+---
+fixes:
+  - |
+    Fixed the api documentation for :class:`.RZXGate` which was incoherent with its notation.
+    The default notation for RZX gate was in big-endian and the example shown was in little-endian format.
+    Fix is done in the example section where the gate notation is concatenated with q_0 , q_1 and 
+    big-endian format is followed throughout. 
+    For example intial notation was :
```
Comment (atharva-satpute) [inline releasenotes/notes/incoherence-notation-rzxgate-16284-df215ab4903f5e3e.yaml]:
```suggestion
    For example initial notation was :
```

File: releasenotes/notes/incoherence-notation-rzxgate-16284-df215ab4903f5e3e.yaml
Comment (Cryoris) [inline releasenotes/notes/incoherence-notation-rzxgate-16284-df215ab4903f5e3e.yaml:1]: We don't typically write release notes for doc fixes, you can just delete this file

**Review by Cryoris — COMMENTED** [review]
Thanks for the fix! 

As side note, I'm not sure this notation of `<gate> q0 q1 = <some matrix>` is slightly abusive notation and we might want to fix this at some point (not in this PR!)

File: releasenotes/notes/incoherence-notation-rzxgate-16284-df215ab4903f5e3e.yaml
Comment (Ameya28Bhave) [reply releasenotes/notes/incoherence-notation-rzxgate-16284-df215ab4903f5e3e.yaml:1]: Will do. Thanks for the review @Cryoris

**Review by Shivansh20128 — COMMENTED** [review]
I think its fine now. Congrats on your first issue!

**Comment by Cryoris** [comment]
@Shivansh20128 the release note is not yet deleted, that's still outstanding 🙂

**Comment by Shivansh20128** [comment]
> @Shivansh20128 the release note is not yet deleted, that's still outstanding 🙂

Oh yeah right. Sorry for the oversight.

**Review by Cryoris — DISMISSED** [review]

**Review by Cryoris — APPROVED** [review]

**Pull request merged**